plugins/webapi/mana/logsHandler.go

```diff
@@ -0,0 +1,86 @@
+package mana
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/iotaledger/goshimmer/packages/mana"
+	manaPlugin "github.com/iotaledger/goshimmer/plugins/mana"
+	"github.com/iotaledger/hive.go/identity"
+	"github.com/labstack/echo"
+	"github.com/mr-tron/base58"
+)
+
+// getEventLogsHandler handles the request.
+func getEventLogsHandler(c echo.Context) error {
+	var req GetEventLogsRequest
+	if err := c.Bind(&req); err != nil {
+		return c.JSON(http.StatusBadRequest, GetEventLogsResponse{Error: err.Error()})
+	}
+	var nodeIDs []identity.ID
+	for _, nodeID := range req.NodeIDs {
+		_nodeID, err := mana.IDFromStr(nodeID)
+		if err != nil {
+			return c.JSON(http.StatusBadRequest, GetEventLogsResponse{Error: err.Error()})
+		}
+		nodeIDs = append(nodeIDs, _nodeID)
+	}
+	startTime := time.Unix(req.StartTime, 0)
+	endTime := time.Unix(req.EndTime, 0)
+	var epoch time.Time
```

Comment:
I don't think this small detail matters. I simply want the oldest time. `0001` or `1970` will do.

Reply:
It is not a small detail from API and usability point of view. Would you expect your email client to give you your emails only up to `1970` if you want to fetch all your mails?

Reply:
> So if you don't specify a start/end time, current implementation gives you back events until 1970-01-01 00:00:00 +0000 UTC.

Because this is what happens now with mana events...

Reply:
I don't think it really matters though. Practically no event goes past 2020. :)